src/thunderbots/software/ai/hl/stp/action/pivot_action.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "ai/hl/stp/action/action.h"
+#include "geom/angle.h"
+#include "geom/point.h"
+
+/**
+ * The PivotAction makes the robot pivot from the given pivot point to the given
+ * angle with the given radius
+ */
+class PivotAction : public Action
+{
+   public:
+    /**
+     * Creates a new PivotAction
+     */
+    explicit PivotAction();
+
+    /**
+     * Returns the next Intent this PivotAction wants to run, given the parameters.
+     *
+     * @param robot the robot that should perform the pivot
+     * @param pivot_point the point around which the robot pivots
+     * @param final_angle the absolute, not relative, final angle
+     * @param pivot_radius the radius to maintain while pivoting
```

Comment:
Maybe add here that the robot pivots outside at the edge of the radius. My first assumption is that the center of the robot would be located at the pivot radius

Reply:
I think the center of the robot _is_ at the pivot radius? The `pivot_radius` is used directly in the `PivotIntent` which puts the center of the robot at the pivot radius.

Reply:
Ah I see, you are definitely right. I did this review at the end of a 13 hour workday lol. Please ignore the comment